server.go

```diff
@@ -10,6 +10,8 @@ import (
 	"os"
```

Comment:
I feel like this kind of cleanup is needed in this project, but I feel it should be done by you, as this kind of things tend to be heavily based on personal preferences.